prow/prowjobs/GoogleContainerTools/kpt-config-sync/kpt-config-sync-periodics.yaml

```diff
@@ -41,6 +41,48 @@ prow_ignored:
       # This job requires 8vCPUs or less, so it is "small".
       cloud.google.com/gke-nodepool: small-job-pool
 
+- &config-sync-ci-job-v2
```

Comment:
maybe add a comment about how v2 is different?

Reply:
added a comment